### PR TITLE
feat(template): adjusts basic styling

### PR DESCRIPTION
**Pull request by PatrickDohn**
Summary: 
  * adjust styling
  * updates example JSON with a summary field

### CODE REVIEW

**Review by davidholyko — APPROVED** [review]
LGTM

File: src/components/routes/Routes.js [hunk 1]
```diff
@@ -6,7 +6,9 @@ import {
   NotFoundPage,
   CrispPage,
   CascadePage,
+  GoogleDocPage,
 } from 'pages';
+// import GoogleDocPage from '../pages/GoogleDocPage';
```
Comment (davidholyko) [inline src/components/routes/Routes.js:11]: rm

File: src/components/templates/GoogleDocTemplate/Heading.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { Text, View, StyleSheet } from 'react-pdf';
+import data from '../../../example-json/john_smith.json';
+// import _ from 'lodash';
```
Comment (davidholyko) [inline src/components/templates/GoogleDocTemplate/Heading.js:4]: rm unused

**Review by davidholyko — CHANGES_REQUESTED** [review]
Needs to merge chore first

File: src/components/pages/MahomesPage.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { Mahomes } from 'templates/MahomesTemplate/MahomesTemplate';
+
+export const MahomesPage = () => {
+  return (
+    <div className="googleDoc-page">
```
Comment (PatrickDohn) [inline src/components/pages/MahomesPage.js:6]: Change class names as well. make sure its kabob case (mahomes-page)

**Pull request closed without merging**